crates/anvil/tests/it/fork.rs

```diff
@@ -60,6 +60,21 @@ pub fn fork_config() -> NodeConfig {
         .silent()
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn test_fork_gas_limit_applied_from_config() {
+    let (api, _handle) = spawn(fork_config().with_gas_limit(Some(10_000_000_u128))).await;
+
+    assert_eq!(api.gas_limit(), uint!(10_000_000_U256));
+}
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_fork_gas_limit_disabled_from_config() {
+    let (api, _handle) = spawn(fork_config().disable_block_gas_limit(true)).await;
+
+    // see https://github.com/foundry-rs/foundry/pull/8933
+    assert_eq!(api.gas_limit(), U256::from(U64::MAX));
```

Comment:
can we test mining some blocks here, I just want to rule out any issues with base fee

Reply:
I added just sending a tx, not sure if thats sufficient